### PR TITLE
Remove depend tag not needed

### PR DESCRIPTION
**Pull request by Blast545**
Addresses this regression: http://build.ros2.org/view/Rci/job/Rci__nightly-debug_ubuntu_focal_amd64/23/

Caused by https://github.com/ros2/ros2/pull/985

`mimick` is not a valid `rosdep`, and not needed in the vendor package.

Signed-off-by: Jorge Perez <jjperez@ekumenlabs.com>

### CODE REVIEW

**Comment by Blast545** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/11607/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6745/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/9489/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/11569/)

**Review by hidmic — APPROVED** [review]
LGTM pending green CI

**Comment by clalancette** [comment]
While we are in here, we should also probably fix up the `<maintainer>` tag.  I'd suggest either @Blast545 or @hidmic for now.

**Comment by dirk-thomas** [comment]
And the version number should be 0.0.0 since no release has been tagged yet.

**Comment by Blast545** [comment]
Addressed both of your comments, I think this is ready to be merged.

**Review by dirk-thomas — APPROVED** [review]

**Pull request merged**
